ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,306 @@
+        eval "curl -so /etc/elasticsearch/certs/search-guard-tlstool-1.8.zip https://maven.search-guard.com/search-guard-tlstool/1.8/search-guard-tlstool-1.8.zip --max-time 300 "
+        eval "unzip search-guard-tlstool-1.8.zip -d searchguard "
+        eval "curl -so /etc/elasticsearch/certs/searchguard/search-guard.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/searchguard/search-guard-aio.yml --max-time 300 "
+        eval "chmod +x searchguard/tools/sgtlstool.sh "
+        eval "./searchguard/tools/sgtlstool.sh -c ./searchguard/search-guard.yml -ca -crt -t /etc/elasticsearch/certs/ $debug "
+        if [  "$?" != 0  ]
+        then
+            echo "Error: certificates were not created"
+            exit 1;
+        else
+            logger "Certificates created"
+        fi
+        eval "rm /etc/elasticsearch/certs/client-certificates.readme /etc/elasticsearch/certs/elasticsearch_elasticsearch_config_snippet.yml search-guard-tlstool-1.7.zip -f $debug"
+
+        # Configure JVM options for Elasticsearch
+        ram_gb=$(free -g | awk '/^Mem:/{print $2}')
+        ram=$(( ${ram_gb} / 2 ))
+
+        if [ ${ram} -eq "0" ]; then
+            ram=1;
+        fi
+        eval "sed -i "s/-Xms1g/-Xms${ram}g/" /etc/elasticsearch/jvm.options $debug"
+        eval "sed -i "s/-Xmx1g/-Xmx${ram}g/" /etc/elasticsearch/jvm.options $debug"
+        jv=$(java -version 2>&1 | grep -o -m1 '1.8.0' )
+        if [ "$jv" == "1.8.0" ]
+        then
+            ln -s /usr/lib/jvm/java-1.8.0/lib/tools.jar /usr/share/elasticsearch/lib/
+            echo "root hard nproc 4096" >> /etc/security/limits.conf
+            echo "root soft nproc 4096" >> /etc/security/limits.conf
+            echo "elasticsearch hard nproc 4096" >> /etc/security/limits.conf
+            echo "elasticsearch soft nproc 4096" >> /etc/security/limits.conf
+            echo "bootstrap.system_call_filter: false" >> /etc/elasticsearch/elasticsearch.yml
+        fi
+
+        # Start Elasticsearch
+        startService "elasticsearch"
+        echo "Initializing Elasticsearch..."
+        until $(curl -XGET https://localhost:9200/ -uadmin:admin -k --max-time 120 --silent --output /dev/null); do
+            echo -ne $char
+            sleep 10
+        done
+
+        eval "cd /usr/share/elasticsearch/plugins/opendistro_security/tools/ $debug"
+        eval "./securityadmin.sh -cd ../securityconfig/ -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key $debug"
+
+        echo "Done"
+    fi
+}
+
+## Filebeat
+installFilebeat() {
+    logger "Installing Filebeat..."
+
+    #eval "echo -e '[elasticsearch-7.x]\nname=Elasticsearch repository for 7.x packages\nbaseurl=https://artifacts.elastic.co/packages/7.x/yum\ngpgcheck=1\ngpgkey=https://artifacts.elastic.co/GPG-KEY-elasticsearch\nenabled=1\nautorefresh=1\ntype=rpm-md' | tee /etc/yum.repos.d/elastic.repo"
```

Comment:
If this is not necessary, remove it.

Reply:
Done in ee6bd7426583bb2f823f33004f4e2d05ddcb2dd7